.ci/openvino-onnx/Jenkinsfile

```
@@ -1,7 +1,9 @@
-
 // Copyright (C) 2018-2020 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 
+DOCKER_CONTAINER_NAME="jenkins_ngraph-onnx_ci_${BUILD_NUMBER}"
+DOCKER_IMAGE_TAG = "openvino-onnx-ci"
```

Perhaps we want the image to containt the build number too?
```suggestion
DOCKER_IMAGE_TAG = "openvino-onnx-ci:${BUILD_NUMBER}"
```